html2text/__init__.py

```diff
@@ -803,6 +803,9 @@ def o(self, data, puredata=0, force=0):
             self.outcount += 1
 
     def handle_data(self, data, entity_char=False):
+        if not data:
```

Comment:
Would be better to add a comment here explaining why we're returning and stopping the rest of function.

Reply:
Good idea. I've added a comment in the latest revision.